<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IRepell Programmer</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="MainTab">
    <attribute name="title">
     <string>Flashing</string>
    </attribute>
    <widget class="QTextBrowser" name="DebugWindow">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>256</width>
       <height>481</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ausgabe</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="Counter">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>530</y>
       <width>64</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>540</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Zähler</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FlashButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Programmieren</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="auto_label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wird automatisch  ausgeführt</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="StatusGraphic">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>221</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="FlashProgress">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>221</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SettingsTab">
    <attribute name="title">
     <string>Einstellungen</string>
    </attribute>
    <widget class="QCheckBox" name="AutoFlash">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>361</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Automatisches Flashing beim einlegen der Platine</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QKeySequenceEdit" name="SetFlashHotkey">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>281</width>
       <height>24</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hotkey für Flashing</string>
     </property>
    </widget>
    <widget class="QComboBox" name="FlashChooser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Beide</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nur MCU</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nur Telit</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Was wird geflasht?</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MCUPathBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>281</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MCU Hex Datei</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BrowseMCUFile">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>380</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Durchsuchen</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TelitPathBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>450</y>
       <width>281</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BrowseTelitFile">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>450</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Durchsuchen</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Telit Bin Datei</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BrowserIPECMD">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>520</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Durchsuchen</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IPECMDPathBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>281</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="IPECMDPath">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>500</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IPECMD Pfad</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ClearDebug">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Löschen des Output Fensters vor jedem Flash</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="FullMCUOutput">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Voller Output von der MCU</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
